dnsmasq-plan/config/dnsmasq.conf

```diff
@@ -0,0 +1 @@
+server=/fooxample.com/192.168.0.1
```

Comment:
This hardcodes the `fooxample.com` into the service configuration. It might be a good idea to consider pushing this out to a `default.toml` configuration file, and templating the primary configuration.

Reply:
There are a _ton_ of dnsmasq options for the `dnsmasq.conf` file, but for the meantime I did just default the server configurations in a `default.toml` to example values and added the proper templating syntax to the `dnsmasq.conf` file for server values.

Reply:
For reference, the config provided with a default install on ubuntu has no options. Its a huge file with everything commented.

Reply:
Ok, I'll just add the entire commented file to this plan.